lib/rules.js

```diff
@@ -237,7 +243,9 @@ const hyphensOmittedOrReplacedWithSpaces = new FunctionRule(
 
     // We also want to accept spaces in `test` in place of hyphens in `target`.
     const targetWithSpacesInsteadOfHyphens = target.replace(re, ' ');
-    return [targetWithSpacesInsteadOfHyphens, test];
+    const testWithSpacesInsteadOfHyphens = test.replace(re, ' ');
+
+    return [targetWithSpacesInsteadOfHyphens, bidirectional ? testWithSpacesInsteadOfHyphens: test];
```

Comment:
Do we want to update this rule to be bidirectional? Not sure if there is a specific use case where doing the bidirectional check would give us a bad false positive.

Reply:
I think this one should stay unidirectional. Product specifically asked to not accept something like "good-night" for "good night" in the vocab quiz.